#! /bin/sh /usr/share/dpatch/dpatch-run
## 25-autoconf-fix-ACLOCAL_AMFLAGS.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## DP: do not regenerate ACLOCAL_AMFLAGS

@DPATCH@
--- ACE_wrappers.orig/MPC/modules/AutomakeWorkspaceCreator.pm
+++ ACE_wrappers/MPC/modules/AutomakeWorkspaceCreator.pm
@@ -585,9 +585,9 @@
     my($m4inc) = '-I m4';
     print $fh $crlf,
               'ACLOCAL = @ACLOCAL@', $crlf,
-              'ACLOCAL_AMFLAGS = ',
-              (defined $wsHelper ?
-                 $wsHelper->modify_value('amflags', $m4inc) : $m4inc), $crlf,
+	      #'ACLOCAL_AMFLAGS = ',
+	      #(defined $wsHelper ?
+	      #   $wsHelper->modify_value('amflags', $m4inc) : $m4inc), $crlf,
               'AUTOMAKE_OPTIONS = foreign', $crlf, $crlf,
               (defined $wsHelper ?
                  $wsHelper->modify_value('extra', '') : '');
